libpermian/plugins/kickstart_test/__init__.py

```diff
@@ -230,14 +230,25 @@ def execute(self):
             LOGGER.info(f"Architecture {self.arch} is not supported, skipping the execution.")
             return
 
+        command = self.runner_command
+
+        platforms = self.crcList.by_configuration('platform')
```

Comment:
crcs with different platforms can be split to separate instances of the workflow in factory and this problem can be avoided, or am I missing something?